rpm/python.spec

```diff
@@ -110,6 +111,7 @@ color editor (pynche), and a python gettext program (pygettext.py).
 Summary:    The libraries and header files needed for Python development
 Group:      Development/Libraries
 Requires:   %{name} = %{version}-%{release}
+BuildRequires:  pkgconfig(libcrypt)
```

Comment:
It's already required above: no need to add it twice for the same spec.

Reply:
Good catch that was a typo.